ext/standard/tests/general_functions/proc_open_redirect.phpt

```diff
@@ -4,26 +4,31 @@ Redirection support in proc_open
 <?php
 
 $php = getenv('TEST_PHP_EXECUTABLE');
+$args = getenv('TEST_PHP_EXTRA_ARGS');
+$cmd = "$php $args";
+
 try {
-    proc_open([$php], [['redirect']], $pipes);
```

Comment:
isn't this changing test from array syntax to shell syntax? If so, those are different things to test so it should not be changed. You should rather create array from `$args` and pass it here.

Reply:
I agree, but how do we create an array from `$args`? Am I overlooking an easy solution?

Reply:
Since this seems to have stalled, the stated purpose of this file is not to test the _syntax_ of `proc_open()`, but instead to test that redirection works. While it does test the array syntax as a side effect, that's not the crucial part; so in my opinion it's reasonable to change the syntax to fix this unrelated issue.